Description: Add manpage to systemd service documentation
Author: Valentin Vidic <Valentin.Vidic@CARNet.hr>
Last-Update: 2018-04-25
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/logd/logd.service.in
+++ b/logd/logd.service.in
@@ -1,5 +1,6 @@
 [Unit]
 Description=ha_logd logging daemon
+Documentation=man:ha_logd(8)
 Before=pacemaker.service
 PartOf=pacemaker.service
 
